shell/components/nav/WindowManager/ContainerLogs.vue

```diff
@@ -347,14 +321,17 @@ export default {
     },
 
     updateFollowing() {
-      const el = this.$refs.body;
+      const virtualList = this.$refs.virtualList;
 
-      this.isFollowing = el.scrollTop + el.clientHeight + 2 >= el.scrollHeight;
+      if (virtualList) {
+        this.isFollowing = virtualList.getScrollSize() - virtualList.getClientSize() === virtualList.getOffset();
```

Comment:
Nitpicking here 🙂 

I think grouping with parentheses makes the expression easier to read 

```suggestion
        this.isFollowing = (virtualList.getScrollSize() - virtualList.getClientSize()) === virtualList.getOffset();
```